lib/moonshot/tools/asg_rollout.rb

```diff
@@ -65,6 +65,10 @@ def wait_for_in_service(new_instance)
 
           loop do
             instance_health = asg.instance_health(new_instance)
+            if instance_health.out_of_service?
```

Comment:
I'm not entirely against that. I just fear if people trigger it, and than leave it there and wait for it to finish. And then forget about it, come back to it to see it still running after 30, or so minutes. 

But thinking about this as I type, it's basically the same if they leave it now, it would just quit. So.. Yeah. We can do that. But is there merit in it? Like.. do you think if the first one wasn't able to attach the next one would because the problem the first one saw would go away? 

I mean, when I was testing this, it couldn't create a new instance because we exceeded volumes. So any next one which would have tried to join, wouldn't have succeeded either. 

I'm just brainstorming here. :)